cmd/system-probe/modules/network_tracer.go

```diff
@@ -300,6 +295,7 @@ func writeConnections(w http.ResponseWriter, marshaler encoding.Marshaler, cs *n
 
 	w.Header().Set("Content-type", marshaler.ContentType())
 	w.Write(buf) //nolint:errcheck
+	pool.Put(bs)
```

Comment:
You want to `Put(buf)` here, not `bs`. Otherwise pool.Get() will always return a 0-sized slice with a large underlying buffer